dailymotion-player-objc/DMPlayerViewController.m

```diff
@@ -30,9 +32,22 @@ - (void)openURLInSafari:(NSURL *)URL;
 @implementation DMPlayerViewController
 
 - (void)dealloc {
-  UIWebView *webView = (UIWebView *)self.view;
-  webView.delegate = nil;
```

Comment:
@JanGorman Looks like it's very specific case: I cannot reproduce it and only 1% users meets this crash.